lib/definitions/cloud-build-service.d.ts

```diff
@@ -76,6 +76,13 @@ interface ICloudBuildService {
 		projectId: string,
 		androidBuildData?: IAndroidBuildData,
 		iOSBuildData?: IIOSBuildData): Promise<void>;
+
+	/**
+	 * Returns the path to the directory where the build output may be found.
+	 * @param {ICloudBuildOutputDirectoryOptions} options Options that are used to determine the build output directory.
+	 * @returns {string} The build output directory.
+	 */
+	getCloudBuildOutputDirectory(options: ICloudBuildOutputDirectoryOptions): string;
```

Comment:
It is in the cloud build service, so we can rename it to `getBuildOutputDirectory`